test/run-e2e-ocp.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+set -e -u -o pipefail
+
+declare PROJECT_ROOT
+PROJECT_ROOT="$(git rev-parse --show-toplevel)"
+declare OPERATORS_NS
+OPERATORS_NS="openshift-operators"
+declare NO_INSTALL=false
+declare NO_UNINSTALL=false
+declare SHOW_USAGE=false
+
+install_obo_OCP(){
+  oc apply -f ./hack/olm/catalog-src.yaml
+  oc apply -f ./hack/olm/subscription.yaml
+}
+
+wait_for_operators_ready(){
+  local ret=0
+  local -i wait_time=0
+  oc -n openshift-operators rollout status deploy/obo-prometheus-operator || ret=1
+  while [[ ret -eq 1 ]]; do
+    if [[ wait_time -lt 60 ]];
+    then
+      ret=0
+    else
+      echo "Failed to install OBO"
+      exit 1
```

Comment:
scripts should try to have a single exit point. So it is better to use `return` so that main can fail / handle it.